tests/operator_test.go

```diff
@@ -2943,6 +2945,109 @@ spec:
 			}, time.Minute*5, time.Second*2).Should(BeTrue())
 		})
 	})
+
+	FContext("[Serial] Seccomp configuration", Serial, func() {
+
+		Context("install policy", func() {
+			var nodeName string
+
+			const expectedSeccompProfilePath = "/proc/1/root/var/lib/kubelet/seccomp/kubevirt/kubevirt.json"
+
+			enableInstall := func(enable bool) {
+				By(fmt.Sprintf("Configuring install policy to %t", enable))
+				kv := util2.GetCurrentKv(virtClient)
+				kv.Spec.Configuration.SeccompConfiguration = &v1.SeccompConfiguration{
+					InstallPolicy: enable,
+				}
+				tests.UpdateKubeVirtConfigValueAndWait(kv.Spec.Configuration)
+			}
+
+			restartHandler := func() {
+				kvNamespace := util2.GetCurrentKv(virtClient).Namespace
+
+				By(fmt.Sprintf("Restart handler on node %s", nodeName))
+				pod, err := kubecli.NewVirtHandlerClient(virtClient).Namespace(kvNamespace).ForNode(nodeName).Pod()
+				Expect(err).NotTo(HaveOccurred())
+				Expect(pod.Namespace).ToNot(BeEmpty(), kvNamespace)
+
+				err = virtClient.CoreV1().Pods(pod.Namespace).Delete(context.TODO(), pod.Name, metav1.DeleteOptions{})
+				Expect(err).NotTo(HaveOccurred())
+
+				Eventually(matcher.ThisPod(pod), 30*time.Second, time.Second).Should(BeGone())
+
+				// Wait for new handler to be functional
+				Eventually(func() error {
+					_, err := tests.ExecuteCommandInVirtHandlerPod(nodeName, []string{"/usr/bin/ls"})
+					return err
+				}, 1*time.Minute, time.Second).ShouldNot(HaveOccurred())
+			}
+
+			BeforeEach(func() {
+				nodeName = libnode.GetAllSchedulableNodes(virtClient).Items[0].Name
+
+				By("Removing profile if present")
+				_, err := tests.ExecuteCommandInVirtHandlerPod(nodeName, []string{"/usr/bin/rm", "-f", expectedSeccompProfilePath})
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("should install custom policy", func() {
+				enableInstall(true)
+
+				restartHandler()
```

Comment:
Do we require a restart now? You can use a config-change callback to install it without handler restart if that is the case.

Reply:
I did take this approach as I was not sure how we should react in case the feature would be disabled. Should we remove the profile in such a case? Would the user expect it? And finally, should we take some precautions to determine if the profile is provided by us or by admin(and therefore we should not touch it)